.github/workflows/cd.yml

```diff
@@ -3,7 +3,14 @@ name: Deploy
 on:
   # !! Uncomment the following lines once you've set up the dev environment and ready to turn on continuous deployment
   # push:
-  #   branches: ["main"]
+  #   branches: 
+  #     - 'main'
+  #   paths:
+  #     - 'app/**'
+  #     - 'bin/**'
+  #     - 'infra/**'
+  #     - 'template-only-bin/**'
+  #     - 'template-only-test/**'
```

Comment:
I think these two lines aren't needed as those directories won't exist outside the template.